exporters/otlp/profiles/src/test/java/io/opentelemetry/exporter/otlp/profiles/ProfilesRequestMarshalerTest.java

```diff
@@ -304,6 +304,8 @@ private static Profile.Builder sampleProfileBuilder() {
                         .AGGREGATION_TEMPORALITY_CUMULATIVE)
                 .build())
         .addAllComment(listOf(8L, 9L))
+        .addStringTable("foo")
+        .addStringTable("bar")
```

Comment:
This adds test coverage for the binary encoding. Making this change on `main` causes tests to fail, but the tests pass on this branch, indicating your fix is successful.